# msg/SensorSignal.msg
uint32 sensor_id
float64 value
builtin_interfaces/Time sent_timestamp
float64 sync_offset_ms
